examples/charts/file/hover_span.py

```diff
@@ -17,16 +17,16 @@
                value_vars=['Biology', 'Business', 'Computer Science'],
                value_name='Count', var_name='Degree')
 
-vline = Line(data, y='Count', color='Degree', title="Lines VLine", ylabel='measures',
+vline = Line(data, x='Year', y='Count', color='Degree', title="Lines VLine", ylabel='measures',
              tools=TOOLS)
 
-hline = Line(data, y='Count', color='Degree', title="Lines HLine",
+hline = Line(data, x='Year', y='Count', color='Degree', title="Lines HLine",
              ylabel='measures', tools=TOOLS)
 
-int_vline = Line(data, y='Count', color='Degree', title="Lines VLine Interp",
+int_vline = Line(data, x='Year', y='Count', color='Degree', title="Lines VLine Interp",
                  ylabel='measures', tools=TOOLS)
 
-int_hline = Line(data, y='Count', color='Degree', title="Lines HLine Interp",
+int_hline = Line(data, x='Year', y='Count', color='Degree', title="Lines HLine Interp",
```

Comment:
These missing `x='Year'` arguments on only some of the plots in this example are the focus of this PR.